modules/web-element.mjs

```diff
@@ -324,22 +324,188 @@ class PaddingLeft extends WebElementFunction {
+     */
+    constructor() {
+        super("background-color");
+    }
+    get(element) {
+        var doc = element.ownerDocument;
+        var win = doc.defaultView ||doc.parentWindow;
+        var elem = doc.querySelector('.testElem')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('background-color')
+        return cssProp;
+    }
+}
+/**
+ * Function that extract margin-top of a DOM
+ */
+class CheckMarginTop extends WebElementFunction{
+    constructor(){
+        super("margin-top")
+    }
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow
+        var elem = doc.querySelector('.testElem')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('margin-top')
+        return cssProp
+    }
+}
+/**
+ * Function that extract margin-bottom of a DOM
+ */
+class CheckMarginBottom extends WebElementFunction{
+    constructor(){
+        super("margin-bottom")
+    }
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow
+        var elem = doc.querySelector('.testElem')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('margin-bottom')
+        return cssProp
+    }
+}
+/**
+ * Function that extract margin-left of a DOM
+ */
+class CheckMarginLeft extends WebElementFunction{
+    constructor(){
+        super("margin-left")
+    }
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow
+        var elem = doc.querySelector('.testElem')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('margin-left')
+        return cssProp
+    }
+}
+/**
+ * Function that extract margin-right of a DOM
+ */
+class CheckMarginRight extends WebElementFunction{
+    constructor(){
+        super("margin-right")
+    }
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow
+        var elem = doc.querySelector('.testElem')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('margin-right')
+        return cssProp
+    }
+}
+/**
+ * Function that extract paddig-top of a DOM
+ */
+class CheckPaddingTop extends WebElementFunction{
+    constructor(){
+        super("padding-top")
+    }
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow;
+        var elem = doc.querySelector('.testElem')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('padding-top')
+        return cssProp
+    }
+}
+/**
+ * Function that extract paddig-bottom of a DOM
+ */
+class CheckPaddingBottom extends WebElementFunction{
+    constructor(){
+        super("padding-bottom")
     }
-    get(e){
-        var s = e.style.fontFamily
-        return s
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow;
```

Comment:
A utility method could probably me made available in `WebElementFunction` (or somewhere else) in order to get the `window` from an element, seeing as most of the WebElementFunctions will need to repeat these two lines.